assets/dev/scss/editor/preview/preview.scss

```diff
@@ -904,12 +904,45 @@ html.elementor-html {
 }
 
 // Muted elements in the editor
-[data-elementor-device-mode="desktop"] .elementor-hidden-desktop,
-[data-elementor-device-mode="tablet"] .elementor-hidden-tablet,
-[data-elementor-device-mode="mobile"] .elementor-hidden-phone {
+
+[data-elementor-device-mode="desktop"] .elementor-section.elementor-hidden-desktop,
+[data-elementor-device-mode="tablet"] .elementor-section.elementor-hidden-tablet,
+[data-elementor-device-mode="mobile"] .elementor-section.elementor-hidden-phone {
+
+	> .elementor-element-overlay {
+		background-color: #ddd;
+		mix-blend-mode: color;
+	}
+
+	&:before {
+		content: '';
+		display: block;
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		background-color: rgba(255, 255, 255, 0.6);
+		z-index: 9997;
+	}
+}
+
+[data-elementor-device-mode="desktop"] .elementor-column.elementor-hidden-desktop,
+[data-elementor-device-mode="tablet"] .elementor-column.elementor-hidden-tablet,
+[data-elementor-device-mode="mobile"] .elementor-column.elementor-hidden-phone {
+	background-color: fade_out($editor-lightest, .2);
+
+	> .elementor-column-wrap {
+		filter: alpha(0.4) saturate(0);
+	}
+}
+
+[data-elementor-device-mode="desktop"] .elementor-widget.elementor-hidden-desktop,
+[data-elementor-device-mode="tablet"] .elementor-widget.elementor-hidden-tablet,
+[data-elementor-device-mode="mobile"] .elementor-widget.elementor-hidden-phone {
 	background-color: fade_out($editor-lightest, .2);
 
 	.elementor-widget-container {
-		filter: alpha(0.2) saturate(0);
+		filter: alpha(0.4) saturate(0);
 	}
```

Comment:
You can merge these selectors by changing them to something like `[data-elementor-device-mode="desktop"] .elementor-hidden-desktop:not(.elementor-section)`